po-sample-web-conference/src/app/speaker/speaker-card/speaker-card.component.html

```diff
@@ -0,0 +1,23 @@
+<po-widget
+  p-height="200"
+  p-primary-label="Lectures"
+  p-secondary-label="Edit"
+  (p-primary-action)="detail()"
+  (p-secondary-action)="edit()">
+
+  <a [routerLink]="['/speakers/detail/', speaker.id]">
+    <po-avatar
+      class="po-xl-4 po-lg-3"
+      p-size="lg"
+      p-src="../../assets/images/{{ speaker.photo }}">
+    </po-avatar>
+  </a>
+
+  <div class="po-xl-8 po-lg-9 card-description">
+    <a class="title-link" [routerLink]="['/speakers/detail/', speaker.id]">
+      <p class="po-font-text-large-bold">{{ speaker.name }}</p>
+    </a>
+    <p class="po-font-text">{{ speaker.email }}</p>
+  </div>
+
+</po-widget>
```

Comment:
A rota da foto e do título não está funcionando, pode ser alterado no seguinte trecho:
```suggestion
<po-widget
  p-height="200"
  p-primary-label="Lectures"
  p-secondary-label="Edit"
  (p-primary-action)="detail()"
  (p-secondary-action)="edit()">

  <a [routerLink]="['detail/', speaker.id]">
    <po-avatar
      class="po-xl-4 po-lg-3"
      p-size="lg"
      p-src="../../assets/images/{{ speaker.photo }}">
    </po-avatar>
  </a>

  <div class="po-xl-8 po-lg-9 card-description">
    <a class="title-link" [routerLink]="['detail/', speaker.id]">
      <p class="po-font-text-large-bold">{{ speaker.name }}</p>
    </a>
    <p class="po-font-text">{{ speaker.email }}</p>
  </div>

</po-widget>
```